api/src/lib.rs

```diff
@@ -244,62 +186,54 @@ impl InternedStringId {
 pub struct CachedInternedStringId {
     value: &'static str,
     #[cfg(target_pointer_width = "32")]
-    interned_string_id_and_context: AtomicU64,
+    interned_string_id: AtomicU32,
     #[cfg(target_pointer_width = "64")]
-    interned_string_id_and_context: Mutex<(usize, usize)>,
+    interned_string_id: AtomicU64,
```

Comment:
Can we just use `AtomicUsize` and remove the conditional compilation?

Reply:
Good call!